addons/mail/static/tests/tours/mail_full_composer_test_tour.js

```diff
@@ -36,7 +36,7 @@ tour.register('mail/static/tests/tours/mail_full_composer_test_tour.js', {
         const messaging = await Component.env.services.messaging.get();
         const uploaders = messaging.models['FileUploader'].all();
         inputFiles(
-            uploaders[0].fileInput,
+            uploaders[1].fileInput,
```

Comment:
This is not very reliable, maybe filer the uploaders depending on the owner or something like that?

Reply:
Here we get two uploaders: one for `ComposerView` and the second one is `Chatter` (as implemented now).

Reply:
So indeed it should be possible to just take the uploader from the chatter record instead of accessing array element.